packages/core/src/internal/data-grid/data-grid-types.ts

```diff
@@ -372,6 +372,7 @@ export type ProvideEditorComponent<T extends InnerGridCell> = React.FunctionComp
     readonly target: Rectangle;
     readonly forceEditMode: boolean;
     readonly isValid?: boolean;
+    readonly theme: Theme;
```

Comment:
I added this to make the theme available to custom editors, but eventually ended up not using this. But I think it could be useful to have access to this when implementing editors. Wdyt?